msfrpcd

```diff
@@ -4,52 +4,161 @@
 # $Id$
 #
 # This user interface listens on a port and provides clients that connect to
-# it with an RPC interface to the Metasploit Framework.
+# it with an RPC or JSON-RPC interface to the Metasploit Framework.
 #
 # $Revision$
 #
 
-msfbase = __FILE__
-while File.symlink?(msfbase)
-  msfbase = File.expand_path(File.readlink(msfbase), File.dirname(msfbase))
+RPC_TYPE = 'Msg'
+
+@localconf = "#{ENV['HOME']}/.msf4"
+@ws_tag = 'msf-ws'
```

Comment:
I'm guessing the answer is "following the precedent of `msfdb`, but why are all of these global variables instead of constants?

Reply:
That is a correct assumption, that said, I could make those changes.